src/pages/patientView/trialMatch/TrialMatchTableUtils.ts

```diff
@@ -85,9 +85,41 @@ export function groupTrialMatchesByAgeNumerical(armGroup: ITrialMatch[]): IClini
         clinicalGroupMatch.notMatches = positiveAndNegativeMatches.notMatches;
         return clinicalGroupMatch;
     });
+    if (matches.length > 1) {
+        return removeDuplicationByAge(matches);
+    }
     return matches;
 }
 
+export function removeDuplicationByAge(clinicalGroupMatch: IClinicalGroupMatch[]): IClinicalGroupMatch[] {
+    const uniqClinicalGroupMatch: IClinicalGroupMatch[] = _.clone(clinicalGroupMatch);
+    const duplicateIndexes: number[] = [];
+    for (let i = 0; i < clinicalGroupMatch.length - 1; i++) {
+        for (let j = i + 1; j < clinicalGroupMatch.length; j++) {
+            if (!_.isUndefined(clinicalGroupMatch[i]) && !_.isUndefined(clinicalGroupMatch[j]) &&
+                isIdenticalArrays(clinicalGroupMatch[i].trialOncotreePrimaryDiagnosis.positive, clinicalGroupMatch[j].trialOncotreePrimaryDiagnosis.positive) &&
+                isIdenticalArrays(clinicalGroupMatch[i].trialOncotreePrimaryDiagnosis.negative, clinicalGroupMatch[j].trialOncotreePrimaryDiagnosis.negative)) {
+                if (!uniqClinicalGroupMatch[i].trialAgeNumerical.includes(clinicalGroupMatch[j].trialAgeNumerical)) {
+                    uniqClinicalGroupMatch[i].trialAgeNumerical += ',' + clinicalGroupMatch[j].trialAgeNumerical;
```

Comment:
i think the name of the function should reflect this little operation here.  we aren't just removing duplicates but merging in some sense.